de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v2_0/CombinedFskPortObject.java

```diff
@@ -702,10 +705,33 @@ public JComponent[] getViews() {
     metaDataPane.setName("Meta data");
 
     final JPanel librariesPanel = UIUtils.createLibrariesPanel(packages);
+    List<String> resourcesList = new ArrayList<>();
+    fillResoucesList(firstFskPortObject, resourcesList);
+    fillResoucesList(secondFskPortObject, resourcesList);
+    final JPanel resourcesPanel = UIUtils.createResourcesPanel(resourcesList);
 
+    
     JPanel simulationsPanel = new SimulationsPanel();
 
-    return new JComponent[] {createScriptPanel(), metaDataPane, librariesPanel, simulationsPanel};
+    return new JComponent[] {createScriptPanel(), metaDataPane, librariesPanel,resourcesPanel , simulationsPanel};
+  }
+  
+  public List<String> fillResoucesList(FskPortObject fskObject , List<String> resourcesList){
+    if(fskObject instanceof CombinedFskPortObject) {
+      fillResoucesList(((CombinedFskPortObject)fskObject).getFirstFskPortObject(), resourcesList);
+      fillResoucesList(((CombinedFskPortObject)fskObject).getSecondFskPortObject(), resourcesList);
+    }else {
+      if (fskObject.getEnvironmentManager().isPresent()) {
+        String modelName = SwaggerUtil.getModelName(fskObject.modelMetadata);
+        String padding = String.join("", Collections.nCopies(25, "-"));
+        resourcesList.add(padding+modelName+padding);
+        
+        List<String> elements = Arrays.asList(((ArchivedEnvironmentManager) fskObject.getEnvironmentManager().get()).getEntries());
```

Comment:
@ahmadswaid Seems to work great, but here (and in FSKPortObject.java) you cast to ArchivedEnvrionmentManager, which throws an error when a model is created using the editor. I suggest to either add cases for the other Manager Classes (FilesEnvironment,AddedFilesEnvironment and ExistingEnvironment) or add a getEntries Method to the Interface (probably more elegant but you decide).